src/vga.js

```diff
@@ -1578,7 +1578,7 @@ VGAScreen.prototype.port3C9_read = function()
     var color = this.vga256_palette[index];
 
     this.dac_color_index_read++;
-    return (color >> (2 - offset) * 8 & 0xFF) / 255 * 63 | 0;
+    return (color >> (2 - offset) * 8 & 0xFF) / 256 * 64 | 0;
```

Comment:
Is this correct though? It should probably be the inverse of the formula on line 1548. I can't find any reference on how the hardware converts 18-bit to 24-bit colours.